<launch>


<node pkg="can_to_ros" type="cut_in_disengager" name="cut_in_disengager" output="screen" required="true"/>
<node pkg="can_to_ros" type="cruise_start_safety_check" name="cruise_start_safety_check" output="screen" required="true" />

<node pkg="can_to_ros" type="cbf_smoother" name="cbf_smoother" output="screen" required="true" />

<node pkg="setpointreader" type="setpointreader_node" name="setpointreader_node" output="screen" required="true" />

<node pkg="velocity_controller" type="velocity_controller_node" name="velocity_controller_node" output="screen" required="true" />



</launch>
